### PR TITLE
Adding files homework two

### PR DESCRIPTION
**Pull request by TamaraVanyan**

### CODE REVIEW

File: SecondTask/AverageAndModulus.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+        System.out.println("Enter the second short number:");
+        short num2 = scanner.nextShort();
+
+        System.out.println("Enter the third short number:");
+        short num3 = scanner.nextShort();
+
+        int average = (num1 + num2 + num3) / 3;
+
+        int sumOfAbsValues = Math.abs(num1) + Math.abs(num2) + Math.abs(num3);
```
Comment (David91) [inline SecondTask/AverageAndModulus.java]: It's prohibited to use ANY Java feature we haven't covered so far. Find other solution for getting absolute values of the numbers.

File: SecondTask/AverageAndModulus.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        double result = (double) sumOfAbsValues / average;
```
Comment (David91) [inline SecondTask/AverageAndModulus.java]: Will the result always be calculated normally? Think about your inputs possible values and make corrections here.

File: SecondTask/EntertainmentVenue.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        int entranceFee = 5000;
+//i am adding some changes for gitHub
+
+        System.out.println("Enter the amount for order:");
+        double orderAmount = scanner.nextDouble();
+        if (isVip) {
+            double discount = (orderAmount * 15)/100;
+            orderAmount = orderAmount - discount;
+        }
+        double totalAmountSpent = entranceFee + orderAmount;
```
Comment (David91) [inline SecondTask/EntertainmentVenue.java:28]: entrance fee exists for non-vip clients only. Based on that, do some corrections.

File: SecondTask/IsTriangle.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+public class IsTriangle {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter the length of the first side of the triangle:");
+        int a = scanner.nextInt();
+        System.out.println("Enter the length of the second side of the triangle:");
+        int b = scanner.nextInt();
+        System.out.println("Enter the length of the third side of the triangle:");
+        int c = scanner.nextInt();
+
```
Comment (David91) [inline SecondTask/IsTriangle.java]: Lack of validations of inputs.

File: SecondTask/ReverseThreeDigitNumber.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+        int reversedNumber = digit4 * 100 + digit3 * 10 + digit2;
+
+        if (reversedNumber < 100 || reversedNumber > 999) {
+            System.out.println("It is not possible to form a valid three-digit number by reversing.");
+            System.exit(2);
+        }
+
+        System.out.println("The reversed number is: " + reversedNumber);
+        if (number < 100 || number > 999) {
```
Comment (David91) [inline SecondTask/ReverseThreeDigitNumber.java]: So you're doing input validation at the end of the code? Really? :D

**Review by David91 — CHANGES_REQUESTED** [review]

File: SecondTask/ReverseThreeDigitNumber.java (hunk 5 above)
Comment (TamaraVanyan) [reply SecondTask/ReverseThreeDigitNumber.java]: literally it shows how I am stupid

**Pull request merged**